### PR TITLE
Rectangle page implementation

### LINKED ISSUE #1: Add page with formula calculation
Example: сalculate rectangle area, perimeter, and diagonal length with length and width as input. 
Calculations start after changing number in the input fields.
Validation of input is necessary.

image

### PR DESCRIPTION
**Pull request by Pagendera**

### CODE REVIEW

**Review by ashotvdovychenko — APPROVED** [review]

File: rectangle-page/rectangle-page.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function calculateRectangle() {
```
Comment (neodavis) [inline rectangle-page/rectangle-page.js:3]: No validation on input, user should see some error text

File: rectangle-page/rectangle-page.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <title>Rectangle calculator</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+
+    <script defer src="./rectangle-page.js"></script>
+</head>
+<body class="bg-primary container text-white">
+<h1>Rectangle Calculator</h1>
+    <div class="my-5">
+        <label for="length">Length:</label>
+        <input type="number" id="length" oninput="calculateRectangle()">
```
Comment (neodavis) [inline rectangle-page/rectangle-page.html]: Avoid oninput, migrate call of method in addEventListener in JS file

File: rectangle-page/rectangle-page.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <label for="width">Width:</label>
+        <input type="number" id="width" oninput="calculateRectangle()">
```
Comment (neodavis) [inline rectangle-page/rectangle-page.html]: add some styles for input (for page in general, avoid plain styled elements)

File: rectangle-page/rectangle-page.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Rectangle calculator</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+
+    <script defer src="./rectangle-page.js"></script>
+</head>
+<body class="bg-primary container text-white">
```
Comment (neodavis) [inline rectangle-page/rectangle-page.html]: as for me, would be better to migrate main content into something like card.

**Review by neodavis — CHANGES_REQUESTED** [review]

**Review by neodavis — APPROVED** [review]

**Pull request merged**